Stellar-transaction.x

```diff
@@ -534,17 +534,31 @@ struct AuthorizedInvocation
     AuthorizedInvocation subInvocations<>;
 };
 
-struct AddressWithNonce
+struct AddressAuthorizer
```

Comment:
Maybe we should rename `Authorizer` to `Authorization`, and rename this to `AddressAuthorization` given it contains the signatures?